bepasty_cli/cli.py

```diff
@@ -38,6 +50,15 @@
     'fname',
     help='Filename for piped input.')
 @click.option(
+    '-m',
```

Comment:
well, using m (and not f) just because the setting is max-foo is not very intuitive.
ok, as the more intuitive (lowercase) options in this case are already taken, -L or -T maybe?